tests/autoblocks/test_tracer.py

```diff
@@ -448,3 +452,82 @@ def test_tracer_start_span(*args, **kwargs):
 
     assert tracer._properties.get("span_id") is None
     assert tracer._properties.get("parent_span_id") is None
+
+
+def test_tracer_prod_evaluations(httpx_mock):
+    test_evaluation_id = uuid.uuid4()
+
+    class MyEvaluator(BaseEventEvaluator):
+        id = "my-evaluator"
+
+        def evaluate_event(self, event: TracerEvent) -> EventEvaluation:
+            return EventEvaluation(
+                evaluator_external_id=self.id,
+                id=test_evaluation_id,
+                score=0.9,
+                threshold=Threshold(gte=0.5),
+            )
+
+    httpx_mock.add_response(
+        url=INGESTION_ENDPOINT,
+        method="POST",
+        status_code=200,
+        json={"traceId": "my-trace-id"},
+        match_headers={"Authorization": "Bearer mock-ingestion-key"},
+        match_content=make_expected_body(
+            dict(
+                message="my-message",
+                traceId="my-trace-id",
+                timestamp=timestamp,
+                properties={
+                    "evaluations": [
+                        {
+                            "evaluatorExternalId": "my-evaluator",
+                            "id": str(test_evaluation_id),
```

Comment:
since this UUID is going to be generated internally and not by the end user, you can test it by mocking the uuid library https://github.com/autoblocksai/python-sdk/blob/3885382637244b262df41c33a34fba4fc39e777e/tests/autoblocks/test_tracer.py#L419-L423